lib/base/RequestClient.d.ts

```diff
@@ -59,9 +59,33 @@ declare namespace RequestClient {
   export interface RequestClientOptions {
     /**
      * A timeout in milliseconds. This will be used as the HTTPS agent's socket
-     * timeout, and as the default request timeout.
+     * timeout, AND as the default request timeout.
      */
     timeout?: number;
+    /**
+     * https.Agent keepAlive option
+     */
+    keepAlive?: boolean;
+    /**
+     * https.Agent keepAliveMSecs option
+     */
+    keepAliveMsecs?: number;
+    /**
+     * https.Agent maxSockets option
+     */
+    maxSockets?: boolean;
+    /**
+     * https.Agent maxTotalSockets option
+     */
+    maxTotalSockets?: boolean;
```

Comment:
Some of these types don't look correct. This one should be a `number`: https://nodejs.org/api/http.html#agentmaxtotalsockets

Reply:
Updated @childish-sambino. However, this created a separate commit which breaks our PR standards. Is there a way I can squash these together? 

Better yet, what is the typical process for addressing feedback to avoid a new commit?

Reply:
Your process was correct. Additional commits during PR review/feedback are fine and expected. The standard is that they result in a single commit when merging. This can be accomplished using the `Squash and merge` option below for merging.